modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-cluster-wide-migration_{context}"]
+= About cluster-wide migration
```

Comment:
For this title, maybe something along the lines of "Control plane configuration migration requirements" is more descriptive of the actual content below? Then you could potentially drop the repeated "Control planes" from all the subheadings.

Reply:
done